pex/jobs.py

```diff
@@ -172,7 +172,7 @@ def __str__(self):
         return str(self._job)
 
 
-_CPU_COUNT = cpu_count()
+_CPU_COUNT = cpu_count() or 2
```

Comment:
Might add a comment here saying "2 was an arbitrary choice, to avoid being blocked by one long-running job", or something.